bin/create_k8s_token.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
```

Comment:
feels a bit strange to have bash and ruby mixed ... but it's not too complicated so meh ...